### DISCUSSION TITLE
Hexo theme and css not working with github custom domain

### DISCUSSION THREAD

**Discussion by leodenglovescode** [hexojs/hexo#4933] [Question / Answer] [answered]
Hi, My theme was working when I uploaded the code to Github, but after I used my custom domain (server.leosblog.xyz), the website loaded nothing except the text, but when not using the custom domain, only the Github one, works without a flaw! Can anyone help me? I desperately need it.

**Comment by yoshinorin** [comment, marked as answer]
closed with https://github.com/hexojs/hexo/issues/4934
